api/Makefile

```diff
@@ -16,10 +16,17 @@ define build-static
 	cd ../web; REACT_APP_KEYCLOAK_AUTH_URL="{{REACT_APP_KEYCLOAK_AUTH_URL}}" REACT_APP_MATOMO_URL="{{REACT_APP_MATOMO_URL}}" REACT_APP_MATOMO_SITE_ID="{{REACT_APP_MATOMO_SITE_ID}}" REACT_APP_MATOMO_CONTAINER="{{REACT_APP_MATOMO_CONTAINER}}" npm run build
 endef
 
+define run-env-canada-model
+	POSTGRES_HOST=localhost $(POETRY_RUN) python -m app.weather_models.env_canada ${1}
+endef
+
 define run-env-canada
-	POSTGRES_HOST=localhost $(POETRY_RUN) python -m app.weather_models.env_canada GDPS
-	POSTGRES_HOST=localhost $(POETRY_RUN) python -m app.weather_models.env_canada HRDPS
-	POSTGRES_HOST=localhost $(POETRY_RUN) python -m app.weather_models.env_canada RDPS
+	${call run-env-canada-model,GDPS}
+	${call run-env-canada-model,HRDPS}
+	${call run-env-canada-model,RDPS}
+	# POSTGRES_HOST=localhost $(POETRY_RUN) python -m app.weather_models.env_canada GDPS
+	# POSTGRES_HOST=localhost $(POETRY_RUN) python -m app.weather_models.env_canada HRDPS
+	# POSTGRES_HOST=localhost $(POETRY_RUN) python -m app.weather_models.env_canada RDPS
```

Comment:
did you mean to leave these commented out?

Reply:
Nope - removed.